gammapy/time/tests/test_exptest.py

```diff
@@ -0,0 +1,21 @@
+import numpy as np
```

Comment:
Please add these two boilerplate lines at the top of `gammapy/time/tests/test_exptest.py`:
```
# Licensed under a 3-clause BSD style license - see LICENSE.rst
from __future__ import absolute_import, division, print_function, unicode_literals
```
We have those in every library and test file in Gammapy.